powerspikegg/frontend/src/app/home/searchbar.component.ts

```diff
@@ -1,15 +1,34 @@
 import { Component, OnInit } from '@angular/core';
+import { QuerySenderService } from '../services/query-sender.service';
```

Comment:
Place your project imports below general modules imports, with one space separating them. For example:

```
import { Component, OnInit } from '@angular/core';

import { QuerySenderService } from '../services/query-sender.service';

/* My code */
```